packages/components/src/styles/meta/meta.stories.ts

```diff
@@ -42,61 +37,57 @@ export const Default = {
 };
 
 /**
- * The sd-meta in all possible combinations of `variant` and `size`.
+ * Meta can be used in 2 colors alternatives:
+ *
+ * - black (default)
+ * - `sd-meta--light`: neutral-700
  */
 
-export const LightAndSize = {
-  name: 'Light × Size',
-  parameters: { controls: { exclude: ['sd-meta--light', 'sd-meta--size-...'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'sd-meta--light' },
-        y: { type: 'attribute', name: 'sd-meta--size-...' }
-      },
-      args
-    });
-  }
+export const Variants = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <time class="sd-meta" datetime="2023-08-11">Black</time>
```

Comment:
I feel this example is misleading because you are defining the `datetime` and at the same overriding with text. Would prefer to either remove the text or change the markup to a span or a div